modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,105 @@
+
+Before you can subscribe to Redfish hardware events for the node, you must create a `BMCEventSubscription` custom resource (CR) for the node.
+
+[IMPORTANT]
+====
+You can only create a `BMCEventSubscription` CR for physical hardware that supports Redfish and has a vendor interface set to `redfish` or `idrac-redfish`.
+====
+
+[NOTE]
+====
+Use the `BMCEventSubscription` CR to subscribe to pre-defined Redfish events. The Redfish standard does not provide an option to create specific alerts and thresholds. For example, to receive an alert event when an enclosure's temperature exceeds 40° Celsius, you need to manually configure the event according to the vendor's recommendations.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Deploy a bare-metal node with Redfish hardware in your cluster.
+* Get the user name and password for the bare-metal node BMC.
+
+.Procedure
+
+. Confirm the hardware has the Redfish `EventService` enabled using the following `curl` command:
++
+[source,terminal]
+----
+curl https://<bmc_ip_address>/redfish/v1/EventService --insecure -H 'Content-Type: application/json' -u "<user_name>:<password>"
+----
++
+where:
++
+--
+bmc_ip_address:: is the IP address of the BMC controller
+--
++
+.Example output
+[source,terminal]
+----
+{
+   "@odata.context":"/redfish/v1/$metadata#EventService.EventService",
+   "@odata.id":"/redfish/v1/EventService",
+   "@odata.type":"#EventService.v1_0_2.EventService",
+   "Actions":{
+      "#EventService.SubmitTestEvent":{
+         "EventType@Redfish.AllowableValues":[
+            "StatusChange",
+            "ResourceUpdated",
+            "ResourceAdded",
+            "ResourceRemoved",
+            "Alert"
+         ],
+         "target":"/redfish/v1/EventService/Actions/EventService.SubmitTestEvent"
+      }
+   }
+...
+}
+----
+
+. Create a `BMCEventSubscription` resource to subscribe to the Redfish events:
+
+.. Save the following YAML in the `bmc_sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BMCEventSubscription
+metadata:
+  name: sub-01
+  namespace: openshift-machine-api
+spec:
+   hostName: <hostname> <1>
+   destination: https://hw-event-proxy-cloud-native-events.apps.compute-1.example.com/webhook <2>
+   context: ''
+----
+<1> Specifies the name or UUID of the worker node where the Redfish events are generated.
+<2> Specifies the Baremetal Hardware Event proxy service.
++
+.. Create the `BMCEventSubscription` CR:
++
+[source,yaml]
+----
+$ oc create -f bmc_sub.yaml
+----
++
+.Example output
+[source,json]
+----
+{
+   "ServiceEnabled":true,
+   "Status":{
+      "Health":"OK",
+      "HealthRollup":"OK",
+      "State":"Enabled"
+   }
+}
```

Comment:
I'm wondering why this part is separate from the other output

Reply:
Will correct this, thanks.